mce-io.c

```diff
@@ -868,6 +856,39 @@ void mce_unregister_io_monitor(gconstpointer io_monitor)
 	return;
 }
 
+gboolean mcs_io_monitor_seek_to_end(gconstpointer io_monitor)
+{
+	iomon_struct *iomon = (iomon_struct *)io_monitor;
+	GError *error = NULL;
+	gboolean seek_success = FALSE;
+	loglevel_t loglevel = io_mon_get_log_level(iomon);
+
+	if ((g_io_channel_get_flags(iomon->iochan) & G_IO_FLAG_IS_SEEKABLE) == G_IO_FLAG_IS_SEEKABLE) {
+		g_io_channel_seek_position(iomon->iochan, 0, G_SEEK_END, &error);
+		if(error)
+			mce_log(loglevel, "%s %s", __func__, error->message);
+		else
+			seek_success = TRUE;
+		g_clear_error(&error);
+	}
+	
+	error = NULL;
+	
+	if(!seek_success) {
+		int bytes_read;
+		char buffer[1024];
+		int count = 0;
+		do {
+			g_io_channel_read_chars(iomon->iochan, buffer,
+						1024,
+						&bytes_read, &error);
+			count++;
```

Comment:
This variable is not used, unless I am missing something.

Reply:
upps yes thats left over form debugging